tests/helpers.py

```diff
@@ -0,0 +1,37 @@
+# Copyright 2016 Ian Cordasco
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Test functions useful across twine's tests."""
+
+import contextlib
+import os
+
+
+@contextlib.contextmanager
+def set_env(**environ):
```

Comment:
pytest's [monkeypatch](https://docs.pytest.org/en/latest/monkeypatch.html) fixture can be used to modify an environment variable for a test - up to you, but it saves an extra helper function.

Reply:
Done